packages/vuepress-theme-odyssey/templates/DocsTemplateComponent.vue

```diff
@@ -7,7 +7,7 @@
       <template v-slot:right>
         <ul v-if="$page.frontmatter.links" class="docs-page-header--links">
           <li v-for="link in $page.frontmatter.links" :key="link.label">
-            <a :href="link.href">
+            <a :href="link.href" target="_blank">
```

Comment:
I'm anti external here, per our own guidelines:

>Use an external link when:
> - The destination of the link aids in the completion of a task on the current tab (e.g. help documentation)
> - Opening the link in the current tab would result in loss of data or task interruption (e.g. completing a long form)

In this case, it would be users choosing to leave our site and visit GH or Figma, right?

Reply:
Correct, I would argue that because documentation is supplemental to GitHub and Figma links, you would want both open at the the same timeline and therefore it aligns with our guideline (first item).

I'll defer to the broader team here though.

Reply:
[gets out my External Links speech 😂]

I think "lemme pop this into a new tab" is def a real world use case for docs, but users already have a way to open links in new tabs provided by the browser. `target="_blank"` is for _forcing_ users to open something in a new tab, which doesn't seem necessary or advantageous here.

@joejezowski-okta thoughts here?

Reply:
Spoke with Joe and Shred offline, removing.